test/functional/forall/tests/test-forall-utils.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef __TEST_FORALL_UTILS_HPP__
-#define __TEST_FORALL_UTILS_HPP__
+#ifndef __TEST_FORALL_SEGMENT_UTILS_HPP__
+#define __TEST_FORALL_SEGMENT_UTILS_HPP__
```

Comment:
I had originally desired the utils header to be dependant of the test group/type, did you want to make individual utils for each group or is this a typo?

Reply:
That was a mistake. It's changed back to what it was.